debian/debian_swraid_lvm_luks.md

```diff
@@ -16,41 +16,45 @@ This guide *could* work at any other provider with a rescue system.
 
 - Boot to the rescue system via hetzners server management page
 - install a minimal Debian 9.4 with hetzners "installimage" skript (https://wiki.hetzner.de/index.php/Installimage)
-- I choosed the following logical volumes on my system to keep it simple:
+- I chose the default raid1/lvm setup to keep it simple:
 
 ```
-lv-home (60GB) ext4
-lv-log (30GB) ext4
-lv-swap (10GB) swap
-lv-root (all) -> means remaining space ext4
+cat configs/simple-debian64-raid-lvm
+installimage -c configs/simple-debian64-raid-lvm
 ```
-To achieve this setup you have to add the following lines
-```
-LV vg0 home /home ext4 60G
-LV vg0 swap swap swap 4G
-LV vg0 log /log ext4 30g
-LV vg0 root / ext4 all
-```
-And change the partition section accordingly
-```
-PART /boot ext3 512M
-PART lvm vg0 all
-```
-- after you adjusted all parameters in the install config file, press F10 to install the debian minimal system
+- let the installation process complete
 - reboot and ssh into your fresh installed system
 
 ### First steps on your fresh Debian installation
 
-- connect via ssh-key you choosed before for the rescue image (attention to the .ssh/known_hosts file..)
-- install busybox and dropbear
-- `apt update && apt install busybox dropbear lvm2`
-- Edit your `/etc/initramfs-tools/initramfs.conf` and set `BUSYBOX=y`
-- Create a new ssh key for unlocking your encrypted volumes when it is rebooting
-- `ssh-keygen -t rsa -b 4096 -f .ssh/dropbear`
-- Create the needed folders for dropbear keys
-- run `vi /etc/dropbear-initramfs/authorized_keys`
-- Paste your pub key `.ssh/dropbear.pub` in there
+- connect via ssh-key you chose before for the rescue image (attention to the .ssh/known_hosts file..)
+- edit `/etc/ssh/sshd_config` and set `#PasswordAuthentication yes` to `PasswordAuthentication no
+- change your root password: `passwd`
+- update and install dropbear-initramfs
+  - `apt update && apt upgrade && apt install dropbear-initramfs`
+- Create a new ssh key on your _local_ computer for unlocking your encrypted volumes when it is rebooting
+  - `ssh-keygen -t ecdsa -f .ssh/id_dropbear`
+- Create a new ssh config for the unlocking process
+  - edit on your _local_ computer: `.ssh/config`:
+  
+ ```
+ host myserver-unlock
+   HostName <IP Adress of your Server>
+   IdentityFile ~/.ssh/id_dropbear
+   User root
+   UserKnownHostsFile ~/.ssh/known_hosts.initramfs
+ ```
+ 
+- Back on the server create the needed folders for dropbear keys
+- Create and edit `/etc/dropbear-initramfs/authorized_keys`:
+
+```
+no-port-forwarding,no-agent-forwarding,no-X11-forwarding,command="/bin/cryptroot-unlock" <paste the public key of your local computer here (.ssh/id_dropbear)>
```

Comment:
Maybe a hint where the user is supposed to find the _local_ public-key is useful here

Reply:
so something like `(most likely in your home folder under .ssh/id_dropbear.pub)`

Also I messed up big here and forgot the .pub. Maybe add a warning to never paste the private key anywhere?